CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Code conventions
+
+* use black to lint the code
+* default values for Part/Assembly config:
+  * defaults for common hyper-parameters (e.g. number of layers): no
+  * defaults for rarely/never tuned hyper-parameters (e.g. optimizer constants): maybe
+  * defaults for structural elements (parts / assemblies / pyTorch modules/functions): yes
```

Comment:
```suggestion
  * defaults for structural elements (parts / assemblies / PyTorch modules/functions): yes
```
I think this is the convention regarding capitalization, see e.g. the [official website](https://pytorch.org/).